spec/definitions/Scheduling/ExportScheduleInstruction.yaml

```diff
@@ -0,0 +1,13 @@
+description: The calculation instruction of scheduled time
```

Comment:
This isn't the most clear description. Maybe something more like "The definition of the recurring schedule"